### PR TITLE
percona-server@5.5 5.5.54-38.7

### PR DESCRIPTION
**Pull request by nijikon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by nijikon** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
```
dyld: lazy symbol binding failed: Symbol not found: _clock_gettime
  Referenced from: /private/tmp/percona-server@5.5-20170323-86632-p20lgo/percona-server-5.5.54-38.7/extra/./comp_err
  Expected in: /usr/lib/libSystem.B.dylib
```

**Comment by fxcoudert** [comment]
Should be moved to `std_cmake_args` like https://github.com/Homebrew/homebrew-core/pull/11522

File: Formula/percona-server@5.5.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class PerconaServerAT55 < Formula
   desc "Drop-in MySQL replacement"
-  homepage "https://www.percona.com/"
-  url "https://www.percona.com/downloads/Percona-Server-5.5/Percona-Server-5.5.41-37.0/source/tarball/percona-server-5.5.41-37.0.tar.gz"
-  version "5.5.41-37.0"
-  sha256 "4de65ccbdd6c266f18339c2ea5427a15d90a8ce1ce1c7574aa2e72f685a10833"
-  revision 1
+  homepage "https://www.percona.com"
```
Comment (vszakats) [inline Formula/percona-server@5.5.rb]: Nit of the nit, but is it intentional to remove the `/` from the bare domain? (I added it a commit ago)

Comment (nijikon) [inline Formula/percona-server@5.5.rb]: It was removed by mistake, so can re-add it. Just curious why is it so important to have a `/` at the end?

Comment (vszakats) [inline Formula/percona-server@5.5.rb]: As a general good practice for forming URLs. If not there, f.e. the browser or curl will have to add it.

**Comment by nijikon** [comment]
@fxcoudert satisfied?

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Great, thanks!

**Pull request closed without merging**